---
layout: post
title:  "Introducing the Microbiome Working Group"
date:   2016-11-30 08:08:27 +0100
categories: jekyll update
---

At McMaster University, I am lucky to be part of the Surette laboratory, which has many microbiome-focussed projects and personal. However, with the growing use of the 16S rRNA gene sequencing pipeline that I developed, we begun to realize how many other labs at McMaster were using microbiota datasets to help answer their research questions. This meant that often one or two researchers per-lab were studying and using microbiome-related tools in almost perfect isolation.
 
In April 2015, Dr. Jennifer Stearns and I launched [the Microbiome Working Group][working]. This Group continues today as a weekly, drop-in work/discussion hour with the aim of bringing microbiome researchers together to teach and learn from one another. That's the official tagline, anyway, but to honest: it was really as simple as wanting colleagues in this field to get to know each other! (You work with who you know, right?) In the last year and a half, we've seen this group develop into a small-but-mighty team of microbiome researchers interested in tackling everything from typos in R code to the greater philosophical ideas behind what we're studying. Its been particularly rewarding to witness the growth of some of our "regulars" who started out fearing that a mis-command could delete every file on their system to absolutely falling in love with the command line.
 
This was the conceptual idea that Jen and I had behind the Working Group from the beginning: For better or worse, academia is a constant cycling of trainees, and there can be large gaps in knowledge that leave a research group with a graduate once they have their degree in hand. The Working Group was a way to mitigate this. It was a way for the learner to become the teacher, and for our collective knowledge to remain available to the next generation of trainees.

[working]: http://hmjournalclub.wordpress.com/microbiome-working-group/
